airflow/dags/transform_warehouse/dbt_test.yml

```diff
@@ -7,8 +7,11 @@ cmds:
 arguments:
   - '/app/scripts/run_and_upload.py'
   - 'run'
```

Comment:
just noting that it is kind of confusing that the command here is `run` but 🤷‍♀️

Reply:
Yeah, I'll think of a new name if we ever revisit this